app/src/main/java/com/xmartlabs/template/BaseProjectApplication.java

```diff
@@ -57,6 +58,8 @@ private void initializeInjections() {
     bullet = createBullet(component);
     Injector.getInstance().setObjectGraph(bullet);
     Injector.inject(this);
+    UIInjector.getInstance().setObjectGraph(bullet);
```

Comment:
That will still have the problem of having to inject both every time.
What about doing all in the `Injector` of the core (that maybe handle both/any object graphs) and a single inject does everything?